docs/configuration.md

```diff
@@ -59,6 +59,8 @@ Relative paths specified within the config file are relative to the config file
 
 - <a name="deprecateTypingAliases"></a> **deprecateTypingAliases** [boolean]: PEP 585 indicates that aliases to types in standard collections that were introduced solely to support generics are deprecated as of Python 3.9. This switch controls whether these are treated as deprecated. This applies only when pythonVersion is 3.9 or newer. The default value for this setting is `false` but may be switched to `true` in the future.
 
+- <a name="identifyUnreachableCode"></a> **identifyUnreachableCode** [boolean]: If enabled, code that is determined to be unreachable by type analysis is reported using a tagged hint. This setting does not affect code that is determined to be unreachable regardless of type analysis. such code is always reported as such. This setting also has no effect when when using the command-line version of pyright because it never emits tagged hints for unreachable code.
```

Comment:
> regardless of type analysis. such code is always reported as such.

There's either a punctution or capitalization issue here. Also overuse of the word "such" IMO. :)
```suggestion
- <a name="identifyUnreachableCode"></a> **identifyUnreachableCode** [boolean]: If enabled, code that is determined to be unreachable by type analysis is reported using a tagged hint. This setting does not affect code that is determined to be unreachable regardless of type analysis; such code is always reported as unreachable. This setting also has no effect when when using the command-line version of pyright because it never emits tagged hints for unreachable code.
```